app.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var app = express();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
```

Comment:
The connection string is incorrect - it should be 'mongodb://localhost/foodTruckAPI'